### ISSUE TITLE
Do not shows tweets with "\" symbol

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#33]
```
Do not shows tweets with "\" symbol. Error in Regex replacing.
```

Original issue reported on code.google.com by `tro...@gmail.com` on 9 Jan 2010 at 12:44

**Comment by GoogleCodeExporter** [comment]
```
Does not recognizes "strange" symbols in the status or in the nicknames
```

Original comment by `elg...@gmail.com` on 12 Mar 2010 at 4:50

**Comment by GoogleCodeExporter** [comment]
```
Closed by 6c146697d675 revision.
```

Original comment by `tro...@gmail.com` on 25 Mar 2010 at 11:22
- Changed state: **Fixed**
